test/unit/models/container.js

```diff
@@ -101,6 +102,37 @@ describe('Container', function () {
           })
       })
   })
+
+  describe('stats', function () {
+    it('should start and get stats stream', function (done) {
+      var stream = require('stream')
+      var res = new stream.Transform({
+        transform: function (data, encoding, cb) {
+          cb(null, data)
+        }
+      })
+      res.writeHead = sinon.spy()
+      res.on('data', function (data) {
+        container.stop().then(function () {
+          container.onDelete()
+        })
+      })
+      res.on('end', function () {
+        assert(res.writeHead.calledOnce)
+        done()
+      })
+
+      container.getStats({query: {stream: '1'}}, res)
+      return assert.isFulfilled(container.start())
+    })
```

Comment:
for this test, you may want to consider using [sinon timers](http://sinonjs.org/docs/#clock). Since you have built in a 1000 ms delay for sending logs, it makes this test take over 2 seconds... I don't know if that's the issue you're hitting (timeout causing the error), but it's where I would start